e2e/interchaintestv8/e2esuite/suite.go

```diff
@@ -68,17 +105,22 @@ func (s *TestSuite) SetupSuite(ctx context.Context) {
 	}))
 
 	// map all query request types to their gRPC method paths for cosmos chains
-	s.Require().NoError(populateQueryReqToPath(ctx, s.ChainB))
+	s.Require().NoError(PopulateQueryReqToPath(ctx, s.ChainB))
+
+	if s.ethTestnetType == TestnetTypePoW {
+		anvil := chains[1].(*icethereum.EthereumChain)
+		faucet, err := crypto.ToECDSA(ethcommon.FromHex("0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"))
```

Comment:
Shouldn't have faucet address be hard coded this way. At lease use the global variable.

Reply:
Put it in a var for now, will look into consolidating these